containers/codespaces-linux/test-project/test.sh

```diff
@@ -31,6 +33,18 @@ check "pydocstyle" pydocstyle --version
 check "bandit" bandit --version
 check "virtualenv" virtualenv --version
 
+# Check Python packages
+check "numpy" python -c 'import numpy'
```

Comment:
I'm not sure what's wrong here. I've been testing these changes by copying the Dockerfile and scripts to a test codespace and rebuilding its container. Everything works as expected there:

![Python test commands work in a codespace](https://user-images.githubusercontent.com/19893438/166618394-264c1aa2-1c45-40db-9bfd-5199d8250be7.png)

I noticed that one of the earlier checks, `pip3 --version` has different output than it does on my codespace:

![Wrong output in smoke test](https://user-images.githubusercontent.com/19893438/166618473-3e8663a1-bc3c-4a76-a783-41445c019793.png)
![Correct output in codespace](https://user-images.githubusercontent.com/19893438/166618455-f959a5f9-6a33-4fd2-9b4c-088a297de596.png)

Python gets confused when `$PATH` isn't set properly. Would the shell running these tests have the same `$PATH` as [we define in `base.Dockerfile`](https://github.com/jungaretti/vscode-dev-containers/blob/28f084fae7252a088ad61cce53304c078ae7d3fe/containers/codespaces-linux/.devcontainer/base.Dockerfile#L37)?